2023/Fortune_Obadiah/README.md

```diff
@@ -105,3 +105,6 @@ With this checklist, we expect you to demonstrate an ability to do research and
 
 ## Second  Blog Post(Acessibility Features Docuentation)
 [Unlocking Digital Inclusion: The Quest for Accessibility](https://fortuneweb3info.medium.com/unlocking-digital-inclusion-the-quest-for-accessibility-a0a36e99e4b6)
+
+## Third Blog Post(My Experience As A Technical Writer)
+[Cultivating Understanding: How Technical Writing Enhances Clarity](https://fortuneweb3info.medium.com/cultivating-understanding-how-technical-writing-enhances-clarity-cfd3e77696eb)
```

Comment:
This is good but I think you might as well add links to the different articles you mention?